d/data-quality/data-quality_ubuntu_16.04.sh

```diff
@@ -0,0 +1,29 @@
+# ----------------------------------------------------------------------------
+#
+# Package		: data-quality
+# Version		: 8.0.10, 8.3.1
+# Source repo	: https://github.com/Talend/data-quality.git
+# Tested on		: ubuntu_16.04
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Nageswara Rao K<nagesh4193@gmail.com>/Priya Seth<sethp@us.ibm.com>
+#
+# Disclaimer: This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+# !/bin/bash
+
+# Install dependencies
+sudo apt-get update -y
+sudo apt-get install -y openjdk-8-jdk maven git  
+
+export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-ppc64el
+export JRE_HOME=${JAVA_HOME}/jre
+export PATH=${JAVA_HOME}/bin:$PATH
+
+git clone https://github.com/Talend/data-quality.git
```

Comment:
Can u pl. add a command line argument to accept the non-default version/tag number and build the corresponding jar

Reply:
I have updated the build script with latest tag and the build is success.

[INFO] Results:
[INFO]
[INFO] Tests run: 18, Failures: 0, Errors: 0, Skipped: 0
[INFO]
[INFO]
[INFO] --- maven-bundle-plugin:2.5.3:bundle (default-bundle) @ dataquality-text-japanese ---
[WARNING] Bundle org.talend.dataquality:dataquality-text-japanese:bundle:8.0.10 : Export org.talend.dataquality.jp.tokenization,  has 1,  private references [com.atilika.kuromoji],
[INFO]
[INFO] --- maven-source-plugin:3.0.1:jar-no-fork (attach-sources) @ dataquality-text-japanese ---
[INFO] Building jar: /root/data-quality/dataquality-text-japanese/target/dataquality-text-japanese-8.0.10-sources.jar
[INFO]
[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ dataquality-text-japanese ---
[INFO] Installing /root/data-quality/dataquality-text-japanese/target/dataquality-text-japanese-8.0.10.jar to /root/.m2/repository/org/talend/dataquality/dataquality-text-japanese/8.0.10/dataquality-text-japanese-8.0.10.jar
[INFO] Installing /root/data-quality/dataquality-text-japanese/pom.xml to /root/.m2/repository/org/talend/dataquality/dataquality-text-japanese/8.0.10/dataquality-text-japanese-8.0.10.pom
[INFO] Installing /root/data-quality/dataquality-text-japanese/target/dataquality-text-japanese-8.0.10-sources.jar to /root/.m2/repository/org/talend/dataquality/dataquality-text-japanese/8.0.10/dataquality-text-japanese-8.0.10-sources.jar
[INFO]
[INFO] --- maven-bundle-plugin:2.5.3:install (default-install) @ dataquality-text-japanese ---
[INFO] Installing org/talend/dataquality/dataquality-text-japanese/8.0.10/dataquality-text-japanese-8.0.10.jar
[INFO] Writing OBR metadata
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] dataquality-parent ................................. SUCCESS [  5.676 s]
[INFO] dataquality-libraries .............................. SUCCESS [01:14 min]
[INFO] dataquality-libraries-devops ....................... SUCCESS [ 39.111 s]
[INFO] dataquality-common ................................. SUCCESS [ 21.750 s]
[INFO] dataquality-record-linkage ......................... SUCCESS [ 18.859 s]
[INFO] dataquality-phone .................................. SUCCESS [  4.191 s]
[INFO] dataquality-sampling ............................... SUCCESS [  3.378 s]
[INFO] dataquality-standardization ........................ SUCCESS [ 37.896 s]
[INFO] dataquality-statistics ............................. SUCCESS [ 26.706 s]
[INFO] dataquality-wordnet ................................ SUCCESS [  6.240 s]
[INFO] dataquality-email .................................. SUCCESS [  2.364 s]
[INFO] dataquality-survivorship ........................... SUCCESS [ 21.664 s]
[INFO] dataquality-converters ............................. SUCCESS [ 17.916 s]
[INFO] dataquality-text-japanese .......................... SUCCESS [ 10.254 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 05:49 min
[INFO] Finished at: 2021-04-19T11:33:52+00:00
[INFO] Final Memory: 58M/345M
[INFO] ------------------------------------------------------------------------